hivemq-edge/src/frontend/src/api/hooks/useGetMetrics/useGetMetrics.tsx

```diff
@@ -6,8 +6,11 @@ import { ApiError, MetricList } from '@/api/__generated__'
 export const useGetMetrics = () => {
   const appClient = useHttpClient()
 
-  return useQuery<MetricList, ApiError>([QUERY_KEYS.METRICS], async () => {
-    const items = await appClient.metrics.getMetrics()
-    return items
+  return useQuery<MetricList, ApiError>({
+    queryKey: [QUERY_KEYS.METRICS],
+    queryFn: async () => {
+      const items = await appClient.metrics.getMetrics()
+      return items
```

Comment:
suggestion(non-blocking): can be returned directly and aligns with other implementation within the PR

```suggestion
      return await appClient.metrics.getMetrics()
```